### PR TITLE
Adds "Ghost Replacement" verb

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Adds "Ghost Replacement" verb to OOC, which allows living players to
request that their mob is taken over by a ghost. Ghosts may not always
be willing or able. Remember that normal rules about metaknowledge are still
applicable.
/:cl:

All the parts are in place, it's just the execution.

### CODE REVIEW

**Comment by calzilla1** [comment]
aaaaaaaaaa...AAAAAAAAAAAAAAAAAAAAAAAA

**Comment by Cyberboss** [comment]
> availible

Love this change, but Kor's gonna lynch you

**Comment by coiax** [comment]
fixed

**Comment by Niknakflak** [comment]
:-1: 
Why trust players to do something that admins are going to have to babysit instead of just having admins do it and cut out the babysit part.  Its a one click button

**Comment by PKPenguin321** [comment]
God damn robots are terkin ma jerb

**Comment by Incoming5643** [comment]
Does this generate a message in attack logs, it needs to if it doesn't.

Also there should probably be some pretty harsh limits on when this can be used.

**Comment by PKPenguin321** [comment]
Yeah now that I think about it maybe have this run a message through admins that allows them to approve or deny it, don't want people just spamming this
And it should also be clearly logged in the mobs VV

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -909,4 +909,12 @@
 					  "[C] topples over [src]!", \
 					  "[C] leaps out of [src]'s way!")]</span>")
 	C.Weaken(2)
-	
\ No newline at end of file
+
+/mob/living/verb/request_replacement()
+	set name = "Ghost Replacement"
+	set category = "IC"
```
Comment (PKPenguin321) [inline code/modules/mob/living/living.dm]: OOC

**Comment by KorPhaeron** [comment]
Interesting idea but feature freeze

**Comment by lzimann** [comment]
I think this might be good as antag-only thing.

**Comment by factoryman942** [comment]
make it not doable if you're restrained or unconscious, like suicide

**Comment by coiax** [comment]
@ChangelingRain reopen plz

**Comment by coiax** [comment]
People who might want to use this feature:

- heads of staff
- silicons
- antagonists who don't feel like antagonising
- anyone who wants/needs to leave

**Comment by lzimann** [comment]
should only go for antags

**Comment by coiax** [comment]
See above.

**Comment by GunHog** [comment]
Very useful feature for someone like me who might have to leave suddenly. I have had to give up Ninja roles and other super fun things. With this, I could play until I have to go, then offer myself up without hoping an admin is here to help.

**Comment by KorPhaeron** [comment]
@coiax 

Get headmin approval please, then let me know

**Comment by coiax** [comment]
Could just tag it with DNM

**Comment by KorPhaeron** [comment]
the pr list is getting long and that upsets me for reasons that don't actually make any sense

**Pull request closed without merging**